src/integration/pages/idxPage/canonicalTests.js

```diff
@@ -0,0 +1,47 @@
+import {
+  runCoreCanonicalTests,
+  runCanonicalAnalyticsTests,
+} from '../../common';
+
+export default () => {
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+
+  it('I can see a radio schedule component with an id', () => {
+    const hasRadioSchedule = service === 'persian';
+    const id = document.getElementById('Radio-Schedule');
+
+    if (hasRadioSchedule) {
+      expect(id).toBeInTheDocument();
+    } else {
+      expect(id).not.toBeInTheDocument();
+    }
+  });
+
+  describe('Story Promo', () => {
+    it('I can see an image', () => {
+      const section = document.querySelector('section');
```

Comment:
What if you were to move this outside of this, and into the `describe` section (so you don't have to duplicate in the image & headline tests?
```
  describe('Story Promo', () => {
      const section = document.querySelector('section');
      
      if (section) {
        it('I can see an image', () => {
          ...
          };
          
        it('I can see a headline', () => {
          ...
      };
      }
```